plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2672,19 +2673,36 @@ private static boolean exists(TrinoFileSystem fs, Location location)
     }
 
     @Override
-    public void createView(ConnectorSession session, SchemaTableName viewName, ConnectorViewDefinition definition, boolean replace)
+    public void createView(ConnectorSession session, SchemaTableName viewName, ConnectorViewDefinition definition, Map<String, Object> viewProperties, boolean replace)
```

Comment:
wrap?

Reply:
Currently it is wrapped in a single line right ? Or do we need to move each arguments to a new line ?

Reply:
i meant the line is long, splitting across lines would be appropriate